salt/modules/file.py

```diff
@@ -3747,7 +3747,7 @@ def rmdir(path):
         return exc.strerror
 
 
-def remove(path):
+def remove(path, **kwargs):
```

Comment:
This isn't your code, but why `kwargs` and not explicit keyword arguments?

Reply:
What's the reason of passing the force option that is ignored in `file.remove`?

Reply:
FWIW, force can also be passed to the states: `file.absent` and `file.tidied` which are also merrily ignored :trollface: